build-system/specifications.md

```diff
@@ -44,7 +44,11 @@ installation directory which defaults to `$(TOP)`, the `<top>` directory.
 For base, extensions, and IOC applications, the default value can be
 changed in the `configure/CONFIG_SITE` file. The installation directory
 for the EPICS components is controlled by the definition of
-`INSTALL_LOCATION`
+`INSTALL_LOCATION` variable.
```

Comment:
Either leave the original or add `the`.

Reply:
Yeah. Copied the sentence from TeX source and missed 'the'.